### PR TITLE
osconfig_test: copy autogen proto and gapic code for use with tests

### PR DESCRIPTION
**Pull request by adjackura**

### CODE REVIEW

File: osconfig_tests/utils/utils.go [hunk 1]
```diff
@@ -93,6 +93,7 @@ EOM`
 // InstallOSConfigDeb installs the osconfig agent on deb based systems.
 func InstallOSConfigDeb() string {
 	return fmt.Sprintf(`systemctl stop google-osconfig-agent
+dpkg --configure -a
```
Comment (iamsubratp) [inline osconfig_tests/utils/utils.go]: why this?

Comment (adjackura) [reply osconfig_tests/utils/utils.go]: This is a hack to fix the service restart from breaking dpkg, I'm working on a packaging fix to have more garceful shutdowns of the service.

**Pull request merged**